health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MapChart.js

```
@@ -0,0 +1,196 @@
+  const data = Digit.SessionStorage.get(key);
+  const startDate = data?.range?.startDate ? new Date(data?.range?.startDate) : Digit.Utils.dss.getDefaultFinacialYear().startDate;
+  const endDate = data?.range?.endDate ? new Date(data?.range?.endDate) : Digit.Utils.dss.getDefaultFinacialYear().endDate;
+  const title = `${format(startDate, "MMM d, yyyy")} - ${format(endDate, "MMM d, yyyy")}`;
+  const interval = getDuration(startDate, endDate);
+  const denomination = data?.denomination || "Lac";
+  const tenantId = data?.filters?.tenantId || [];
+  return { startDate, endDate, title, interval, denomination, tenantId };
+};
+
+const getColor = (current) => {
+  const status = current && current.status;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)